### ISSUE TITLE
Cannot compile with Go 1.23

### ISSUE THREAD

**Issue by yorickdowne** [bitlayer-org/bitlayer-l2#28]
#### Issue

When attempting to compile with Go 1.23:
```
72.48 # github.com/ethereum/go-ethereum/cmd/geth
72.48 link: github.com/fjl/memsize: invalid reference to runtime.stopTheWorld
72.58 util.go:48: exit status 1
72.58 exit status 1
72.60 make: *** [Makefile:12: geth] Error 1
```

#### Suggested solution

Go team rejected the memsize author’s proposal to make memsize work with Go 1.23.

Upstream go-ethereum fixed this by removing memsize in this patch: https://github.com/ethereum/go-ethereum/commit/e4675771eda550e7eeb63a8884816982c1980644

**Comment by freddi8e** [comment]
fixed in v1.1.5

**Comment by freddi8e** [comment]
fixed in v1.1.5
